.github/workflows/main.yaml

```diff
@@ -74,6 +82,22 @@ jobs:
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
 
+      - name: Build documentation
+        if: and(${{ matrix.test_examples }}, eq('${{ matrix.os }}', 'ubuntu-latest'))
```

Comment:
It does not work. It is still executed on windows. Nothing I tried is working.

Reply:
Curious if the build_docs job was not enough to cover different cases?

Reply:
@justinchuby : the background is that build_docs was failing (see my PR: https://github.com/microsoft/onnx-script/pull/155 ) ... if you have any suggestions, I would appreciate it. Xavier was trying to figure out the same, but I am not sure what his intentions are with these specific changes. I do know there is another issue fetching experimental packages which is interfering with CI currently (fixed in my PR, but not merged in), which is discussed below, not sure if that's a source of Xavier's issues.

Reply:
Thanks! Taking a look

Reply:
Since the CI did not fail before and does now, I wonder if there are changes in the dependencies we take (sphinx themes etc.) that caused this?

Reply:
I wonder if it relates to [this line](https://github.com/microsoft/onnx-script/blob/3c241a092dbe8f0756f7034540585d293a390141/docs/conf.py#L49) ... the corresponding line in ONNX looks like  [this](https://github.com/onnx/onnx/blob/604af9cb28f63a6b9924237dcb91530649233db9/docs/docsgen/source/conf.py#L45)

Reply:
The second line is better. The first line may work or not depending on the current directory. My point about the first comment is not related to the documentation itself but the fact I tried to avoid generating the documentation on windows to make the CI faster.

Reply:
Regarding doc gen, I tried to not generate documentation at all in `test` jobs and separated them to the `build_docs` job below. Does that fit the need, or are there special things we are doing in the `test` jobs that make us want to generate the docs after tests are run?